tests/models/test_hf_model.py

```diff
@@ -477,6 +477,8 @@ def get_lm_trainer(hf_model,
                    should_save_peft_only: bool = False):
     transformers = pytest.importorskip('transformers')
 
+    torch_less_than_2 = version.parse(torch.__version__) < version.parse('2.1.0')
```

Comment:
can we roll up this var given used in 1 place

Reply:
ok fine :P